modules/tensor_mechanics/test/tests/czm/czm_3DC_load_complex.i

```diff
@@ -0,0 +1,236 @@
+[Mesh]
```

Comment:
Delete this test.  Just having the single test that uses the Action input is sufficient. Duplicate input files just makes things harder to maintain.